package.json

```
@@ -33,7 +33,8 @@
   ],
   "scripts": {
     "test": "npm run lint && metatests test/",
-    "lint": "eslint --ignore-path .gitignore .",
+    "lint": "eslint --ignore-path .gitignore . && prettier --debug-check \"**/*.js\" \"**/*.json\" \"**/*.md\" .eslintrc .prettierrc  --ignore-path .prettierignore",
```

`.prettierignore` is used by default
```suggestion
    "lint": "eslint --ignore-path .gitignore . && prettier --debug-check \"**/*.js\" \"**/*.json\" \"**/*.md\" .eslintrc .prettierrc",
```